helmfile/values/ingress-nginx-probe-ingress.yaml.gotmpl

```
@@ -0,0 +1,7 @@
+ingress:
+  host: {{ .Values.ingressNginx.subDomain }}.{{ .Values.global.baseDomain }}
+  annotations:
+    cert-manager.io/cluster-issuer: {{ .Values.global.issuer }}
+  {{ if and .Values.externalTrafficPolicy.local .Values.externalTrafficPolicy.whitelistRange.wcProbeIngress }}
+    nginx.ingress.kubernetes.io/whitelist-source-range: {{ .Values.externalTrafficPolicy.whitelistRange.wcProbeIngress }}
```

```suggestion
    nginx.ingress.kubernetes.io/whitelist-source-range: {{ .Values.externalTrafficPolicy.whitelistRange.global }}
```

Shouldn't it be this ?